Controller/Heroes/Starlight/Cards/WarpHaloCardController.cs

```diff
@@ -10,7 +10,7 @@ public class WarpHaloCardController : StarlightCardController
         public WarpHaloCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
             SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && c.IsHero && IsNextToConstellation(c), useCardsSuffix: false, singular: "hero target next to a constellation", plural: "hero targets next to a constellation"));
-            SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && c.IsVillain && IsNextToConstellation(c), useCardsSuffix: false, singular: "villain target next to a constellation", plural: "villain targets next to a constellation"));
+            SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && base.IsVillain(c) && IsNextToConstellation(c), useCardsSuffix: false, singular: "villain target next to a constellation", plural: "villain targets next to a constellation"));
```

Comment:
`            SpecialStringMaker.ShowListOfCards(new LinqCardCriteria((Card c) => c.IsTarget && !c.IsHero && IsNextToConstellation(c), useCardsSuffix: false, singular: "non-hero target next to a constellation", plural: "non-hero targets next to a constellation"));
`

This isn't a mistake in the effect, it's a mistake in the SpecialString

Reply:
Committing correction